spec/requests/api/internal/v2/prosecution_cases_spec.rb

```diff
@@ -41,16 +41,16 @@
 
           parameter name: "include", in: :query, required: false, type: :string,
                     schema: {},
-                    description: 'Return defendant, offence and hearing data through a has_many relationship </br>
-                                  eg include=defendants,defendants.offences,defendants.defence_organisation,hearing_summaries'
+                    description: 'Can return defendant and hearing summary data as they both have a relationship with prosecution case. Offences, defence organisation and prosecution case data is related to defendants so in turn can also be included. </br>
+                                  eg include=defendants,hearing_summaries,defendants.offences,'
```

Comment:
Suggested wording: `Include top-level and nested associations for a prosecution case. All top-level and nested associations available for inclusion are listed under the relationships key of the response body. e.g. to include defendants and their offences: include=defendants,defendants.offences.`

Thoughts?

Reply:
I like it!